### ISSUE TITLE
- jackd2 and qjackctl - start minimal environment for music production in Clear Linux -

### ISSUE THREAD

**Issue by ionutnechita** [clearlinux/distribution#587]
Hi,

I should have the next package: jackd2
this helps me to start the minimal system to test jackd.

It provides a basic infrastructure for audio applications to communicate with each other and with audio hardware. Through JACK, users are enabled to build powerful systems for signal processing and music production.

```
in other operating system I have:
 - jackd
 - jackd2
 - jackd2-firewire
 - libconfig++9v5
 - libffado2
 - libqt5x11extras5
 - libxml++2.6-2v5
 - qjackctl
 - jack-tools
 - liblo7
 - liblo-dev
```
Thank you for your support.

**Comment by ahkok** [comment]
Yes, this is definitely something we'd love to be able to offer.

Points of contention: 
- we still want `pulse` to be default, so this is likely a thing that users will have to manually switch their systems to enable `jackd`
- Qt stuff should be optional as well
- I don't think libxml++ is essential for jackd, neither is libconfig++

**Comment by ionutnechita** [comment]
any updates?

**Comment by Jacalz** [comment]
I think this is a great idea but I would like to see fluidsynth and qsynth added in as well 👍
